backend/src/tests/index.integration.js

```diff
@@ -23,15 +23,15 @@ describe("GET /", () => {
       .get("/")
       .expect(200)
       .end(function (err, res) {
-        assert(res.body.msg.includes("Hello from quantum simulator API"));
+        expect(res.body.msg).to.include("Hello from quantum simulator API");
```

Comment:
Instead of explicitly checking for the string (which we may get rid of at some point) might be better to test against status codes